sw/device/lib/arch/device_sim_dv.c

```diff
@@ -21,9 +21,6 @@ const uint64_t kClockFreqUsbHz = 48 * 1000 * 1000;  // 48MHz
 
 const uint64_t kUartBaudrate = 1 * (1 << 20);  // 1Mbps
 
-// No Device Stop Address in our DV simulator.
-const uintptr_t kDeviceStopAddress = 0;
+const uintptr_t kDeviceTestStatusAddress = 0x30000000;
 
-const uintptr_t kDeviceTestStatusAddress = 0x1000fff8;
-
-const uintptr_t kDeviceLogBypassUartAddress = 0x1000fffc;
+const uintptr_t kDeviceLogBypassUartAddress = 0x30000004;
```

Comment:
Please add a comment to both of these saying "Defined in hw/top_earlgrey/dv/env/chip_env_pkg.sv" so I know where to look if something stops working.

Reply:
Done, thanks.